pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -368,6 +368,8 @@ type DatacenterSpecVSphere struct {
 	DefaultDatastore string `json:"datastore"`
 	// The name of the datacenter to use.
 	Datacenter string `json:"datacenter"`
+	// The name of the vSphere cluster to use.
```

Comment:
Can we please add that this is required for the OOT csi driver ?

Reply:
On this level no, it's not possible to differentiate in schema if vSphere is using OOT CSI or in-tree volumes.

Reply:
Not sure what does that mean. But the point behind my comment is, to make sure that we don't overlook this fact(the fact that this is needed for thjs CSI driver) and remove it in future releases!

Reply:
oh, you meant the comment...